NEWS.md

```diff
@@ -1,3 +1,12 @@
+## CHANGES WITH 32:
+
+  * `stages/org.osbuild.grub2`: allow the unified grub config scheme
+    to be configured even in pure UEFI configurations.
+
```

Comment:
Did we not add the runner for fedora 36 as well, and the small bug fix for the `ostree.checkout` input?

Reply:
Ah, I didn't include bug fixes. Want me to redo it?